CHANGES.rst

```diff
@@ -8,6 +8,9 @@ Changelog
 
 .. towncrier release notes start
 
+- Fix buildout configuration
```

Comment:
@rodfersou plone.restapi is a core package and therefore uses towncrier for changelog entries. I usually do not create changelog entries for internal buildout changes though. Therefore I'd just remove those lines.